lib/MiqVm/MiqRhevmVm.rb

```diff
@@ -57,6 +58,10 @@ def getCfg(_snap = nil)
       cfg_hash["#{tag}.devicetype"] = "disk"
       cfg_hash["#{tag}.filename"]   = file_path.to_s
       cfg_hash["#{tag}.format"]     = disk.format
+
+      if storage_obj.storage.type == "ISCSI" && !lv_active?(file_path.to_s)
```

Comment:
Super-mega-minor, but every usage of file_path after line 54 calls .to_s, so maybe it's just better to call to_s once up there and then it doesn't have to be repeated here, nor below, nor on line 59.